auslib/admin/views/forms.py

```diff
@@ -245,6 +245,29 @@ class EditScheduledChangeExistingRuleForm(ScheduledChangeForm, EditRuleForm):
     sc_data_version = IntegerField('sc_data_version', validators=[Required()], widget=HiddenInput())
 
 
+class EditScheduledChangeDeleteRuleForm(ScheduledChangeDeleteRuleForm):
+    sc_data_version = IntegerField('sc_data_version', validators=[Required()], widget=HiddenInput())
+    backgroundRate = NullableStringField('Background Rate', validators=[Optional()])
```

Comment:
I don't think you need to include backgroundRate or any of the other Rule fields in this form - they aren't used for Scheduled Deletions (or even in the UI), right? I think the only needed (and valid) fields for updating a scheduled deletion of a rule are "when" and "sc_data_version"? Even change_type, rule_id, and data_version can't be changed?